plugins/defaultplugins/plugin_api_vpp.go

```diff
@@ -22,99 +22,53 @@ import (
 	"github.com/ligato/vpp-agent/plugins/defaultplugins/l2plugin/bdidx"
 )
 
-// GetSwIfIndexes gives access to mapping of logical names (used in ETCD configuration) to sw_if_index.
-// This mapping is helpful if other plugins need to configure VPP by the Binary API that uses sw_if_index input.
-//
-// Example of is_sw_index lookup by logical name of the port "vswitch_ingres" of the network interface
-//
-//   func Init() error {
-//      swIfIndexes := defaultplugins.GetSwIfIndexes()
-//      swIfIndexes.LookupByName("vswitch_ingres")
-//
-func GetSwIfIndexes() ifaceidx.SwIfIndex {
-	return plugin().swIfIndexes
-}
+// API of VPP Plugin
+type API interface {
 
-// GetSwIfIndexes gives access to mapping of logical names (used in ETCD configuration) to sw_if_index.
-// This mapping is helpful if other plugins need to configure VPP by the Binary API that uses sw_if_index input.
-//
-// Example of is_sw_index lookup by logical name of the port "vswitch_ingres" of the network interface
-//
-//   func Init() error {
-//      swIfIndexes := defaultplugins.GetSwIfIndexes()
-//      swIfIndexes.LookupByName("vswitch_ingres")
-//
-func (plugin *Plugin) GetSwIfIndexes() ifaceidx.SwIfIndex {
-	return plugin.swIfIndexes
-}
+	// DisableResync for one or more VPP plugins. Use in Init() phase.
+	DisableResync(keyPrefix... string)
```

Comment:
Pull request is probably missing this point from the TODO list:

- instead of Resync, provide new API to dump (in the protobuf format) all currently installed ACLs on the VPP (expose from defaultplugins; e.g. DumpACL() (acls []*AccessLists_Acl))